### DISCUSSION TITLE
The coupling of phase field and mechanical stress

### DISCUSSION THREAD

**Discussion by biaogxb** [idaholab/moose#21246] [Q&A Modules: Phase field] [answered]
I just came into contact with moose. I want to use the phase field method to simulate the growth of dendrites and couple the mechanical stress by adding  elastic strain energy . I encountered difficulties in coupling the mechanical stress. Is there a similar example?

And I want to customize kappa as kappa=k0 (1+w*cos（ θ））， among θ is the changing angle between the normal vector of interface and the reference axis, I don't know how to express this θ， Do you have any suggestions?

Thank you!

**Comment by GiudGiud** [comment, marked as answer]
Hello

There are examples in the combined module. 
https://github.com/idaholab/moose/tree/next/modules/combined/examples/phase_field-mechanics

Guillaume

**Reply by biaogxb** [reply]
Thank you!!! I want to customize kappa as kappa=k0 (1+w*cos（ θ））， among θ is the changing angle between the normal vector of interface and the reference axis, I don't know how to express this θ， Do you have any suggestions?

**Reply by GiudGiud** [reply]
so this theta, is it a property of the dendrites or is it local to the mesh?
You should search in moose for models for kappa.

**Reply by biaogxb** [reply]
Okay thank you very much
